### DISCUSSION TITLE
insatll from docker

### DISCUSSION THREAD

**Discussion by Anastasisha** [zitadel/zitadel#7106] [Q&A]
Hello.
Please show a working docker-compose configuration.
ghcr.io/zitadel/zitadel:stable always calls restart container
![zitadel](https://github.com/zitadel/zitadel/assets/154595857/8c413929-d9e2-4848-ba39-dfb9342c0edc)

**Comment by hifabienne** [comment]
hei @stebenz @eliobischof 
Can you help here?

**Comment by eliobischof** [comment]
Which manual did you read?

**Comment by Anastasisha** [comment]
I reed this https://zitadel.com/docs/self-hosting/manage/configure#database-initialization
After restarting the container started working.  Now there is a problem with setting up the smtp connector

**Comment by Anastasisha** [comment]
please tell me what could be the problem?
![Uploading IMG_20231226_160835_695.jpg…]()
